### PR TITLE
Migrate more content

### LINKED ISSUE #256: [Misc] Migrate 63.7% of the docs site
## Description
At the time of creating this ticket, we are migrating `10%` of the content. We would like to bring in more content and verify that we've caught any edge cases.

## Acceptance Criteria
* [ ] Migrate 25% of the content
* [ ] Verify that the site builds correctly
* [ ] Observe build-time impact of pulling in more content

### PR DESCRIPTION
**Pull request by jerelmiller**
Closes #256

## Description

Pulls in more content for the site. This brings us to ~65% of the total content.

### CODE REVIEW

**Review by zstix — DISMISSED** [review]

**Comment by austin-schaefer** [comment]
All I have to say on this is: Woooooooooh let's goooooooooo!

**Comment by roadlittledawn** [comment]
Should we sync Drupal prod DB to dev, commit those changes, then merge this? Or do that during some sustaining time after this MMF?

**Comment by jerelmiller** [comment]
We should be able to merge this once #409 is merged.

File: amplify.yml
Comment (roadlittledawn) [inline amplify.yml:7]: why do we rimraf node_modules folder for amplify?

**Review by zstix — APPROVED** [review]

**Review by roadlittledawn — APPROVED** [review]
looks good. just had one question for my edification

**Pull request merged**
